samples/batch.js

```diff
@@ -16,7 +16,6 @@
 'use strict';
 
 function createBatchTransaction(instanceId, databaseId, projectId) {
-  // [START create_batch_transaction]
```

Comment:
If this region tags is being deleted, and therefore the sample not shown in the docs, we should delete the sample